site/examples/inputs.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const inputWithButton = `<fieldset class='relative'>
+  <button class='btn absolute top right w96 round-r'>Search</button>
+  <input class='input pr96' placeholder='Search'>
```

Comment:
If you type such that you fill up that input, the text is going to be jammed up against the button, right?

Yes, we should avoid *unnecessary* classes; but if this is a pattern that we want people to copy-paste and use all the time, we should not hold back from adding the classes needed to get it right.